paste.c

```diff
@@ -8,7 +8,7 @@ int wmain() {
 	HANDLE clip = GetClipboardData(CF_UNICODETEXT);
 	if (clip) {
 		_setmode(_fileno(stdout), _O_U8TEXT);
-		wprintf(L"%s\n", clip);
+		wprintf(L"%s", clip);
```

Comment:
@Qix- Is it correct to not include the `\n`? I thought all output was supposed to include it.